src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -63,41 +61,41 @@
-                             [0.105, 0.0105, 6.20],    # Md_Mixed
-                             [0.110, 0.0115, 2.80],    # Hd_Mixed
-                             [0.095, 0.0095, 2.50],    # Ld_Residential
-                             [0.100, 0.0115, 6.20],    # Md_Residential
-                             [0.105, 0.0120, 5.00]]),  # Hd_Residential
-    'LoadRatePerv': np.array([[], [], [], [], [], [], [], [], [], [],  # Load Rate on Pervious Surfaces per urban land use per contaminant
-                              [0.015, 0.0021, 0.80],    # Ld_Mixed
-                              [0.015, 0.0021, 0.80],    # Md_Mixed
-                              [0.015, 0.0021, 0.80],    # Hd_Mixed
-                              [0.015, 0.0019, 1.30],    # Ld_Residential
-                              [0.015, 0.0039, 1.10],    # Md_Residential
-                              [0.015, 0.0078, 1.50]]),  # Hd_Residential
-    'DisFract': np.array([[], [], [], [], [], [], [], [], [], [],  # Dissolved Fraction per urban land use per contaminant
-                          [0.33, 0.40, 0],    # Ld_Mixed
-                          [0.33, 0.40, 0],    # Md_Mixed
-                          [0.33, 0.40, 0],    # Hd_Mixed
-                          [0.28, 0.37, 0],    # Ld_Residential
-                          [0.28, 0.37, 0],    # Md_Residential
-                          [0.28, 0.37, 0]]),  # Hd_Residential
-    'UrbBMPRed': np.zeros((16, 3)),  # Urban BMP Reduction
+    'LoadRateImp': [[], [], [], [], [], [], [], [], [], [],  # Load Rate on Impervious Surfaces per urban land use per contaminant
+                    [0.095, 0.0095, 2.80],   # Ld_Mixed
+                    [0.105, 0.0105, 6.20],   # Md_Mixed
+                    [0.110, 0.0115, 2.80],   # Hd_Mixed
+                    [0.095, 0.0095, 2.50],   # Ld_Residential
+                    [0.100, 0.0115, 6.20],   # Md_Residential
+                    [0.105, 0.0120, 5.00]],  # Hd_Residential
+    'LoadRatePerv': [[], [], [], [], [], [], [], [], [], [],  # Load Rate on Pervious Surfaces per urban land use per contaminant
+                     [0.015, 0.0021, 0.80],   # Ld_Mixed
+                     [0.015, 0.0021, 0.80],   # Md_Mixed
+                     [0.015, 0.0021, 0.80],   # Hd_Mixed
+                     [0.015, 0.0019, 1.30],   # Ld_Residential
+                     [0.015, 0.0039, 1.10],   # Md_Residential
+                     [0.015, 0.0078, 1.50]],  # Hd_Residential
+    'DisFract': [[], [], [], [], [], [], [], [], [], [],  # Dissolved Fraction per urban land use per contaminant
+                 [0.33, 0.40, 0],   # Ld_Mixed
+                 [0.33, 0.40, 0],   # Md_Mixed
+                 [0.33, 0.40, 0],   # Hd_Mixed
+                 [0.28, 0.37, 0],   # Ld_Residential
+                 [0.28, 0.37, 0],   # Md_Residential
+                 [0.28, 0.37, 0]],  # Hd_Residential
+    'UrbBMPRed': [[0.0] * 3] * 12,  # Urban BMP Reduction
```

Comment:
I just tried this in the Python REPL and I thought I'd check to make sure it's the behavior you want: it looks like setting it this way causes the elements to be references rather than values in some cases, so when you change `array[0][0]` it also changes `array[1][0]`, `array[2][0]`, and so on. To replicate, open the REPL then try

```python
x = [[0.0] * 3] * 12
x[0][0] = 5
```

The output looks like:

`[[5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0], [5, 0.0, 0.0]]`